packages/pagination/pagination.test.tsx

```diff
@@ -0,0 +1,179 @@
+import React from "react";
+import { shallow, mount } from "enzyme";
+import serializer from "jest-emotion";
+import toJson from "enzyme-to-json";
+
+import Pagination, { getItemCountString } from "./Pagination";
+import PaginationContainer from "./PaginationContainer";
+
+expect.addSnapshotSerializer(serializer);
+
+describe("Pagination", () => {
+  describe("rendering", () => {
+    it("renders default", () => {
+      const component = shallow(
+        <Pagination initialActivePage={4} initialPageLength={20} />
+      );
+
+      expect(toJson(component)).toMatchSnapshot();
+    });
+
+    it("renders wrapped in a PaginationContainer", () => {
+      const component = shallow(
+        <PaginationContainer vertAlignChildren="flex-end">
+          <Pagination initialActivePage={4} initialPageLength={20} />
+        </PaginationContainer>
+      );
+
+      expect(toJson(component)).toMatchSnapshot();
+    });
+
+    it("renders w/ custom item label", () => {
+      const component = shallow(
+        <Pagination
+          itemsLabel="things"
+          initialActivePage={4}
+          initialPageLength={20}
+        />
+      );
+
+      expect(toJson(component)).toMatchSnapshot();
+    });
+
+    it("renders w/ totalItems set", () => {
+      const component = shallow(
+        <Pagination
+          totalItems={200}
+          initialActivePage={4}
+          initialPageLength={20}
+        />
+      );
+
+      expect(toJson(component)).toMatchSnapshot();
+    });
+
+    it("renders w/ totalPages, pageLength, and totalItems set", () => {
+      const component = shallow(
+        <Pagination
+          totalPages={20}
+          pageLength={10}
+          totalItems={200}
+          initialActivePage={4}
+          initialPageLength={20}
+        />
+      );
+
+      expect(toJson(component)).toMatchSnapshot();
+    });
+
+    it("renders prev and next buttons as links", () => {
+      const component = shallow(
+        <Pagination
+          prevUrl="prev"
+          nextUrl="next"
+          initialActivePage={4}
+          initialPageLength={20}
+        />
+      );
+
+      expect(toJson(component)).toMatchSnapshot();
+    });
+
+    it("renders w/ page length menu", () => {
+      const component = shallow(
+        <Pagination
+          showPageLengthMenu={true}
+          initialPageLength={100}
+          initialActivePage={4}
+        />
+      );
+
+      expect(toJson(component)).toMatchSnapshot();
+    });
+  });
+
+  describe("getItemCountString", () => {
+    it("returns {start}-{end} of {total} if all params are passed", () => {
+      expect(getItemCountString(2, "items", 20, 60)).toEqual(
+        "21–40 of 60 items"
```

Comment:
in no way blocking: it would be awesome if library-users could translate that.

Reply:
not sure if the above feedback is sensible anymore.

Reply:
You're right - probably not relevant anymore. I completely agree with you though